ament_cmake_auto/cmake/ament_auto_add_gmock.cmake

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Whitley Software Services, LLC
```

Comment:
@clalancette do copyrights need to be OSRF?

Reply:
No, we don't require that.

However, I will question where this is coming from, given that the copyright is 2021 (on a new file), and the copyright holder doesn't seem to match the contributor.  @jordan-palacios can you tell us where this file came from originally?

Reply:
As stated in https://github.com/ament/ament_cmake/issues/483, this was "inspired" by [ament_auto_add_gtest.cmake](https://github.com/ament/ament_cmake/blob/rolling/ament_cmake_auto/cmake/ament_auto_add_gtest.cmake), though it ended up being somewhat different. 

I can update the year or switch the copyright holder to the company I work for, maintaining the Apache 2.0 license of course. Both work for us. :)

Reply:
> I can update the year or switch the copyright holder to the company I work for, maintaining the Apache 2.0 license of course.

Looking at the two of them, they end up looking pretty different.  So I think it would be best if you update the copyright year and change the copyright holder, either to OSRF or to your company.  Thanks!

Reply:
Done!

Reply:
> Done!

Thanks.  Can you please rebase and add your Signed-off-by: line to that latest commit?  That should make the DCO bot happy.

Reply:
Done!